system/jlib/jevent.hpp

```diff
@@ -196,9 +196,13 @@ class jlib_decl EventRecorder
     CriticalSection cs;
     unsigned sizeMessageHeaderFooter{0};
     unsigned options{0};
+    byte compressionType = 0;
```

Comment:
Not a problem, per se, but seems odd to initialize here and in the constructor initialization list, using different values.

Reply:
Why the different default syntax compared to existing members?

Reply:
I will remove the initializer - the header is not available at this point for the correct value.